setup.cfg

```diff
@@ -22,6 +22,13 @@ description-file = README.md
 author = Apache Airflow PMC
 author-email = dev@airflow.apache.org
 license = Apache License, Version 2.0
+license_files =
+  LICENSE
+  licenses/*
+  NOTICE
+
+[wheel]
```

Comment:
Difference between `[wheel]` and `[bdist_wheel]`?

Reply:
`bdist_wheel` is the command, `wheel` is the section in the config file.

Reply:
`[bdist_wheel]` can be used as section name as well, so I'm a bit confused here. 
Ref: https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels

Reply:
I guess they are aliased. I'll update to use the one in the docs.